packages/pigeon/lib/java_generator.dart

```diff
@@ -765,22 +771,48 @@ Result<$returnType> $resultName =
     });
   }
 
+  void _writeErrorClass(Indent indent) {
+    indent.write('public static class FlutterError extends RuntimeException ');
```

Comment:
This class and its public members all need documentation explaining them, per the style guide.